python/ittapi.native/collection_control.cpp

```diff
@@ -0,0 +1,33 @@
+#include "collection_control.hpp"
+
+#include <ittnotify.h>
+
+
+namespace pyitt
```

Comment:
Need to rename `pyitt` namespace to `ittapi` in all files

Reply:
changed